### DISCUSSION TITLE
point_0700

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#255] [Roy]
"20191221T132231_20191221T132227_T22KHC"
![image](https://user-images.githubusercontent.com/54947195/147529756-35a6b9e4-1655-46c1-94f1-2d80f77ac5c1.png)
![image](https://user-images.githubusercontent.com/54947195/147529803-435a2bc7-7fce-4d03-a92b-39caf31b7313.png)

"20200229T132231_20200229T132228_T22KHC"
![image](https://user-images.githubusercontent.com/54947195/147529934-6dd5874f-9d66-403f-9ba4-f7a4618d9f1c.png)
![image](https://user-images.githubusercontent.com/54947195/147530111-bf4107ff-8ac0-425d-bc16-6fba8cd7a59a.png)

"20200404T132229_20200404T132230_T23KKT"
![image](https://user-images.githubusercontent.com/54947195/147530297-28f88438-82f7-4d31-a5ad-3ac282c88bfc.png)
![image](https://user-images.githubusercontent.com/54947195/147530459-25247a09-b494-4ab8-8eb1-ff7637c92248.png)

@ryali93

**Comment by AntoniCesar** [comment]
"20191221T132231_20191221T132227_T22KHC"
![Captura](https://user-images.githubusercontent.com/86502862/153024487-6db3b12d-2b3b-494d-a45a-3e059a075930.JPG)

**Comment by AntoniCesar** [comment]
"20200229T132231_20200229T132228_T22KHC"
![Captura](https://user-images.githubusercontent.com/86502862/153031985-4f45c0fb-e2a4-4c4a-834f-5ff8350ef596.JPG)
